keanu-python/build.gradle

```diff
@@ -41,10 +36,31 @@ task installDependencies {
     }
 }
 
+task populatePythonClasspaths {
+    dependsOn(":keanu-python:nd4j:copyNd4jJars")
+    dependsOn(":keanu-project:copyJarsIntoPythonClasspath")
+}
+
+task deleteDuplicatedJars() {
```

Comment:
I had tried this with a specific delete task. However, it didn't seem to always honour the `mustRunAfter` and also sometimes deleted the whole of `keanu/classpath`. This other approach works consistently.